package.json

```diff
@@ -25,6 +25,7 @@
     "jwt-decode": "3.1.2",
     "lodash": "4.17.21",
     "moment": "2.29.4",
+    "npm-audit-resolver": "^3.0.0-RC.0",
```

Comment:
`npm add --exact` please - renovate will just raise a PR to pin this dependency by removing the `^` straight away afterwards